### PR TITLE
chore(storybook): set up storybook (#11)

### PR DESCRIPTION
**Pull request by sylvainar**
Set up storybook.

Config is actually quite simple for now, I've configured pretty much everything excepted webpack.

### CODE REVIEW

**Comment by sylvainar** [comment]
I removed the dummy App.spec.jsx so now the build is failing, lol.

File: config/storybook/config.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import * as storybook from '@storybook/react';
+import { setOptions } from '@storybook/addon-options';
+import { withKnobs } from '@storybook/addon-knobs';
+import { setIntlConfig, withIntl } from 'storybook-addon-intl';
+import { addLocaleData } from 'react-intl';
+
+// ================= INTL =================
+import enLocaleData from 'react-intl/locale-data/en';
+import frLocaleData from 'react-intl/locale-data/fr';
+import messages from '../../src/i18n/messages.json';
```
Comment (maxence-lefebvre) [inline config/storybook/config.js]: use `paths.js`

File: config/storybook/config.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+});
+
+// ================= DECORATORS =================
+storybook.addDecorator(withIntl);
+
+storybook.addDecorator(withKnobs);
+
+// ================= STORIES LOADER =================
+const loadStories = () => {
+  const req = require.context('../../src/react/components', true, /\.stories\.jsx$/);
```
Comment (maxence-lefebvre) [inline config/storybook/config.js]: same use `config/paths.js`

File: package.json [hunk 3]
```diff
@@ -36,7 +36,9 @@
     "test": "node scripts/test.js --env=jsdom",
     "extract:messages": "react-intl-cra src/**/*.jsx -o resources/translations/messages.json",
     "extract:pot": "react-intl-po json2pot resources/translations/messages.json -o resources/translations/messages.pot",
-    "extract:po": "react-intl-po po2json resources/translations/*.po -m resources/translations/messages.json -o src/i18n/messages.json"
+    "extract:po": "react-intl-po po2json resources/translations/*.po -m resources/translations/messages.json -o src/i18n/messages.json",
```
Comment (maxence-lefebvre) [inline package.json]: I forgot to upgrade to `-o src/renderer/i18n/messages.json` sorry

**Review by maxence-lefebvre — DISMISSED** [review]

**Review by maxence-lefebvre — APPROVED** [review]

**Pull request merged**